client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx

```diff
@@ -59,7 +62,10 @@ const NewInteractionEventDialog: React.FC<Props> = (props: Props): JSX.Element =
                     בטל
                 </Button>
                 <PrimaryButton
-                    disabled={!canConfirm}
+                    disabled={
+                        !canConfirm || Validator.formValidation(interactionEventDialogData) || 
```

Comment:
Its getting big, maybe you should export it to a function, that we can also use elsewhere